client/app/scripts/superdesk/editor/editor.js

```diff
@@ -199,11 +203,125 @@ function FindReplaceCommand(rootNode) {
     }
 }
 
+function SpellcheckService($q) {
+
+    var selection = document.getSelection();
+
+    this.check = function check(text) {
+        var words = _.uniq(_.filter(text.split(/[\W0-9]+/)));
```

Comment:
But `/[\W0-9]+/` will also match letters like `é`, `ö`, `ñ`, splitting the word `Schön` into `Sch` and `n` - surely this is not desired or ...?